app/views/authenticatedFullUserReactTemplate.scala.html

```diff
@@ -0,0 +1,35 @@
+@import assets.AssetsResolver
+@import com.gu.identity.play.IdUser
+@(title: String, id: String, js: String, user: IdUser)(implicit assets: AssetsResolver)
+
+@scripts = {
+    <script type="text/javascript">
+        window.guardian = window.guardian || {};
+        window.guardian.user = {
+            email: "@user.primaryEmailAddress",
+            @user.publicFields.displayName.map { displayName =>
+            displayName: "@displayName",
+            }
+            @for(fields <- user.privateFields; firstName <- fields.firstName; lastName <- fields.secondName) {
+            firstName: "@firstName",
+            lastName: "@lastName",
+            }
+        };
+    </script>
+    <script type="text/javascript" src="@assets(js)"></script>
+}
+
+@main(title = title, scripts = scripts) {
+    <main id="@id">
+        <noscript>
+            <div style="text-align: center;
+                font-size: 30px;
+                padding-left: 20px;
+                padding-right: 20px;
+                background-color: #e9e939;">
+                Please enable JavaScript - we use it to provide the best experience for Guardian&nbspSupporters.<br/>
```

Comment:
As mentioned above there is a missing semi-colon in the nbsp in this sentence and the whole template looks like it could be dropped in favour of react.scala.html or vice versa.